items.json

```diff
@@ -73396,14 +73396,14 @@
     ]
   },
   {
-    "id": "crpg_heavy_horsemans_mace_v1_h0",
-    "baseId": "crpg_heavy_horsemans_mace_v1",
+    "id": "crpg_heavy_horsemans_mace_v2_h0",
```

Comment:
example of an item that does not need a speed reduction but if wanted, needs to be offset by an increase in other areas to ensure tier does not change too drastically.